python/mxnet/symbol/symbol.py

```diff
@@ -2650,6 +2650,15 @@ def detach(self):
     def backward(self):
         raise NotImplementedForSymbol(self.backward, None)
 
+
+    def _check_dynamic_shape_op(self):
```

Comment:
nit: if you need to make any other changes, consider changing this to "has_dynamic_shape_op"